src/varname.jl

```diff
@@ -348,23 +363,30 @@ julia> vinds(:(x[end]))
 
 julia> vinds(:(x[1, end]))
 :(((1, (lastindex)(x, 2)),))
+
+julia> vinds(:(x[1][end]))
+:(((1,), ((lastindex)(x),)))
```

Comment:
This is incorrect. You want `:(((1,), ((lastindex)(x, 2), )))`, right?

Reply:
This is why I thought it would be difficult to fix btw.

Reply:
Ah :hankey: Maybe it's feasible to instead turn it around and replace the first head of a `:ref` within the `let`s instead.